tidb-cloud/size-your-cluster.md

```diff
@@ -76,15 +109,58 @@ TiDB Cloud deploys TiKV nodes evenly to all availability zones (at least 3) in t
 >
 > When you scale your TiDB cluster, nodes in the 3 availability zones are increased or decreased at the same time. For how to scale in or scale out a TiDB cluster based on your needs, see [Scale Your TiDB Cluster](/tidb-cloud/scale-tidb-cluster.md).
 
-Recommended number of TiKV nodes: `ceil(compressed size of your data ÷ TiKV storage usage ratio ÷ one TiKV capacity) × the number of replicas`
+Although TiKV is mainly used for data storage, the performance of the TiKV node also varies depending on different workloads. Therefore, when planning the number of TiKV nodes, you need to estimate it according to both your [**data volume**](#estimate-tikv-node-quantity-according-to-data-volume) and [performance](#estimate-tikv-node-quantity-according-to-expected-performance), and then take the larger of the two estimates as the recommended node number.
+
+#### Estimate TiKV node quantity according to data volume
+
+You can calculate a recommended number of TiKV nodes according to your data volume as follows:
+
+`node num = ceil(size of your data * TiKV compression ratio * the number of replicas ÷ TiKV storage usage ratio ÷ one TiKV capacity ÷ 3) * 3`
+
+Generally, the usage ratio of TiKV storage is not recommended to exceed 80%, the number of replicas in TiDB Cloud is 3 by default, and the maximum storage capacity of an 8 vCPU, 64 GiB TiKV node is 4096 GiB.
+
+Based on historical data, the average TiKV compression ratio is around 40%.
+
+Suppose that the size of your MySQL dump files is 20 TB and the TiKV compression ratio is 40%. Then, you can calculate a recommended number of TiKV nodes according to your data volume as follows:
+
+ `node num = ceil(20 TB * 40% * 3 ÷ 0.8 ÷ 4096 GiB ÷ 3) * 3 = 9`
+
+#### Estimate TiKV node quantity according to expected performance
```

Comment:
In practice, we found this part sometimes is too optimistic. Sometimes it need quite a few more cores to achieve the task.

Reply:
I added performance data for different latency according to MEXC test data, for both TiDB and TiKV